packages/playground/website/public/puzzle.php

```diff
@@ -0,0 +1,246 @@
+<?php
+error_reporting(0);
+header('Content-Type: application/json');
+
+$open_ai_api_key = getenv('OPEN_AI_API_KEY');
+$api_key = getenv('PUZZLE_API_KEY');
+
+if (!isset($_POST["api_key"])) {
+    response("error", "Please provide an API key.");
+}
+if ($_POST["api_key"] !== $api_key) {
```

Comment:
Where will our users be getting this API key? I apologize if I've missed discussion about this elsewhere.

Reply:
It's [hardcoded here](https://github.com/WordPress/wordpress-playground/pull/1385/files#diff-36e03a13c2f651337a0595e14c5aacbcd9e41ed263b271d973d0b0ef6425016f). 

It's only purpose is to allow us to disable the API in case we start having problems.

Reply:
Thanks! I will send you the details.